Update Template For Http4s 0.18.0-M9

ChristopherDavenport opened the pull request:
Fixes #30

ChristopherDavenport committed:
Update Template For Http4s 0.18.0-M9

ChristopherDavenport committed:
Add sbt-tpolecat and sbt-revolver plugins

rossabaker reviewed (commented):
Is tpolecat too strict for people just getting started? I highly recommend it, but I worry about turning new users off with it. I could argue this either way.

Strong +1 to revolver.

ChristopherDavenport commented:
I can see it both ways as well. I don't like committing code without it personally, but it is definitely detrimental to experimentation and exploration. 

Possible solution. Comment the plugin and leave a note that we recommend it in the README, but that we feel it may not be a reasonable default if you are just learning the interface.

ChristopherDavenport committed:
Comment sbt-tpolecat

Announce in README a description of plugins, adding links to their originating repositories and an explanation of why we use/recommend them.
